build.sbt

```
@@ -121,8 +121,9 @@ javaOptions in Universal ++= Seq(
 
 addCommandAlias("devrun", "run  9200")
 addCommandAlias("prodrun", "run 9200")
-addCommandAlias("fast-test", "testOnly -- -l Acceptance -l WeeklyCheckoutAcceptance -l WeeklyPromoAcceptance")
+addCommandAlias("fast-test", "testOnly -- -l Acceptance -l WeeklyCheckoutAcceptance -l WeeklyPromoAcceptance -l GetAddressIOServiceAcceptance")
 addCommandAlias("acceptance-test", "testOnly acceptance.CheckoutSpec")
```

I think you'll want to make this change too, to ensure that Prout runs these tests (using Travis) after every deploy:

```suggestion
addCommandAlias("acceptance-test", "testOnly acceptance.CheckoutSpec acceptance.GetAddressIOServiceTest")
```

---

updated.